wiki/Storyboard_Scripting/id.md

```diff
@@ -0,0 +1,20 @@
+Storyboard Scripting
+======================
+
+![Sebuah contoh scripting di .osb.](SBS_Base.jpg "Sebuah contoh scripting di .osb.")
+
+  Selain menggunakan [Storyboard Editor](/wiki/Beatmap_Editor/Design)(tab Design) untuk membuat [Storyboards](/wiki/Storyboards) untuk map, Anda juga dapat menggunakan teks editor biasa (seperti Notepad atau [Notepad++](http://www.notepad-plus-plus.org/)). Ini disebut sebagai "**Storyboarding by Scripting**" (SBS), dan memungkinkan Anda untuk lebih tepat (untuk pixel dan milidetik) kapan sprite muncul, apa yang mereka lakukan, dan sebagainya.
```

Comment:
Add a space between `(/wiki/Beatmap_Editor/Design)(tab Design)` (seems to be a mistake in the original article).

`memungkinkan Anda untuk lebih tepat (untuk pixel dan milidetik) kapan sprite muncul` > `membantu Anda agar lebih presisi (hingga satuan pixel dan milidetik) dalam perihal mengatur kapan sprite muncul`